use_cases/examples/word2vec_suspicious_process_directory.conf

```diff
@@ -21,11 +22,11 @@ trigger_focus=word
 trigger_score=center
 trigger_on=low
 trigger_method=stdev
-trigger_sensitivity=6
+trigger_sensitivity=15
 
-outlier_type=process execution
-outlier_reason=suspicious process directory
-outlier_summary=suspicious process directory: {Image}
+outlier_type=TESTING - process execution
```

Comment:
Not sure this is supposed to be here? (TESTING)

Reply:
It's indeed not supposed to be there. Will be removed in next commit.